src/domain/safe/entities/__tests__/multisig-transaction.factory.ts

```diff
@@ -211,9 +212,17 @@ export class MultisigTransactionBuilder
 
   toJson(): unknown {
     const entity = this.build();
+    // TODO: this is exactly the same as the one in ConfirmationBuilder
+    const confirmations = entity.confirmations?.map((confirmation) => {
+      return {
+        ...confirmation,
+        submissionDate: confirmation.submissionDate.toISOString(),
+      };
+    });
```

Comment:
This should be discussed as the `.toJson` is tied to the `Builder` interface.

We need to assume a situation where `MultisigTransactionBuilder` has some actual `Confirmation` set. From the `Confirmation` we cannot call `toJson` since it's not part of that class.